nursery/packaged-as-single-file-dotnet-application.yml

```diff
@@ -0,0 +1,14 @@
+rule:
+  meta:
+    name: packaged as single-file .NET application
+    namespace: executable/installer/dotnet
```

Comment:
thoughts on putting this in `compiler/` next to the other packagers like py2exe, etc?

https://github.com/mandiant/capa-rules/tree/2586016cf2fb92298fb6ac462d88c12c91b76e73/compiler

Reply:
maybe "compiler" isn't the best term there, but thats a different discussion.

Reply:
Hmmm I think it fits better under installer because the .NET code isn't, itself, compiled. It's wrapped by a native bootstrap program that writes all of the necessary files to disk before executing the target assembly. But open to further discussion.